std/conv.d

```diff
@@ -2862,7 +2862,17 @@ unittest
 
     // min and max
     real r = to!real(to!string(real.min_normal));
-    assert(to!string(r) == to!string(real.min_normal));
+    version(NetBSD)
+    {
+        // NetBSD notice
+        // to!string returns 3.3621e-4932L. It is less than real.min_normal and it is subnormal value
+        // Simple C code 
+        //     long double rd = 3.3621e-4932L;
+        //     printf("%Le\n", rd);  
+        // has unexpected result: 1.681050e-4932 
```

Comment:
> Why we think that that after this assert(to!string(r) == to!string(real.min_normal)) is true?

I would expect it to be a bug if converting a real to string and back again didn't result in the same value, though given the kind of funkiness that floating point values tend to have, maybe there isn't a good way around it.

In any case, it seem to me that the correct solution is to either leave in the assertion and let it fail on NetBSD until it's fixed or to do what you've done here but also create a bug report for the issue and reference it here.